endpoints-routes-v1.0.yml

```diff
@@ -107,14 +107,27 @@
   method: GET
     summary: Get list of IGC tracks
     params:
-    - name: uId
```

Comment:
We should be consistent with same name params. I've used uid in the other endpoints, we should stick with one of them